ros2_ws/src/apriltag_ros/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Is this just a copied ros package from somewhere? Do you have the link of the source?

Reply:
Yeah I sourced the apriltag_ros and apriltag_msgs are sourced from a different repo - https://github.com/christianrauch/apriltag_ros
https://github.com/christianrauch/apriltag_msgs